tests/seed_isort_config_test.py

```diff
@@ -51,7 +52,10 @@ def test_integration_isort_cfg(tmpdir):
         assert tmpdir.join('.isort.cfg').read() == expected
 
 
-@pytest.mark.parametrize('filename', ('setup.cfg', 'tox.ini'))
+@pytest.mark.parametrize(
+    'filename',
+    list(filter(lambda cf: cf != '.isort.cfg', SUPPORTED_CONF_FILES)),
```

Comment:
- generally avoid `map` / `filter` / `reduce`, a list comprehension is a more readable alternative.
- [when equivalent, prefer `tuple(...)` to `list(...)`](https://github.com/asottile/scratch/wiki/Prefer-tuples-to-lists)

You can also get the same with `set(SUPPORTED_CONF_FILES) - {'.isort.cfg}`

Reply:
Will change, my (unpopular/weird) opinion is I don't like how `[x for x` looks in list comprehensions 😛 so I tend to write this way